x-pack/solutions/observability/plugins/synthetics/public/apps/locators/monitor_detail.test.ts

```diff
@@ -1,78 +0,0 @@
-/*
```

Comment:
@fkanout Since you kept the logic with `Locator enhancement (covering for time range and tabId)`, I wouldn't remove the monitor_detail.test.ts. Unless there was a specific reason you removed this test.